documentation/modules/configuring/proc-altering-connector-offsets.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-altering-connector-offsets-{context}']
+= Altering connector offsets
+
+[role="_abstract"]
+To alter connector offsets using `KafkaConnector` resources, add the `alterOffsets` configuration.
```

Comment:
This sentence seems to be a bit out of place. You need more than just add the `alterOffsets` field (stop the connector, annotation). So you should either mention all of them or maybe leave out the reference to the `alterOffsets` field?